src/debugger/Debugger.cc

```diff
@@ -1008,11 +1018,14 @@ string Debugger::Cmd::help(std::span<const TclObject> tokens) const
 		"  Note that openMSX comes with a 'disasm' Tcl script that is much "
 		"more convenient to use than this subcommand.";
 	auto disasmBlobHelp =
-		"debug disasm_blob <value> <addr>\n"
+		"debug disasm_blob <value> <addr> <function>\n"
```

Comment:
`<function>` is optional. I think (please check) in other places we then write it as `[<function>]`.